portfolio/src/test/java/FiltersTest.java

```diff
@@ -0,0 +1,69 @@
+import static org.junit.Assert.assertEquals;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.SearchQuery;
+import java.util.Arrays;
+import java.util.List;
+import java.net.MalformedURLException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+public class FiltersTest {
+    @Test
+    public void GeneralFilterTest1() throws MalformedURLException{
```

Comment:
NIT: Use slightly more descriptive test names. If the only difference is special characters in the mockTitle and mockDescription, then specify that in the test name. Alternatively, merge the two test cases.

Reply:
Ok, I think the test cases are quite similar so I'll merge them.